### PR TITLE
ERMS-2241: private Objekte kopieren (Konsortiallizenz)

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: app/grails-app/conf/BootStrap.groovy [hunk 1]
```diff
@@ -87,7 +87,11 @@ class BootStrap {
 
         // def auto_approve_memberships = Setting.findByName('AutoApproveMemberships') ?: new Setting(name: 'AutoApproveMemberships', tp: Setting.CONTENT_TYPE_BOOLEAN, defvalue: 'true', value: 'true').save()
 
-        Setting mailSent = Setting.findByName('MailSentDisabled') ?: new Setting(name: 'MailSentDisabled', tp: Setting.CONTENT_TYPE_BOOLEAN, defvalue: 'false', value: (grailsApplication.config.grails.mail.disabled ?: "false")).save()
+        Setting mailSent = Setting.findByName('MailSentDisabled')
+
+        if(mailSent){
+            mailSent.delete(flush: true)
```
Comment (croqueGrec09) [inline app/grails-app/conf/BootStrap.groovy]: Is flush:true needed?

File: app/grails-app/i18n/messages.properties [hunk 2]
```diff
@@ -3746,7 +3748,7 @@ surveyInfo.startDate.label = Survey start date
 surveyInfo.comment.label = Comments
 surveyInfo.owner.label = Survey owner
 surveyInfo.isMandatory.label =mandatory
-surveyInfo.isMandatory.label.info =Should this survey be a binding survey?
+surveyInfo.isMandatory.label.info =Binding Feedback
```
Comment (croqueGrec09) [inline app/grails-app/i18n/messages.properties]: Mandatory feedback

File: app/grails-app/services/de/laser/SubscriptionService.groovy [hunk 3]
```diff
@@ -87,6 +87,38 @@ class SubscriptionService {
...
+        result
+    }
+
+    List getMySubscriptionsWithMyElements_writeRights(){
+        List result = []
+        List tmpQ
+
+        if(accessService.checkPerm("ORG_INST")) {
+
+            tmpQ = getSubscriptionsConsortialLicenseQuery()
```
Comment (croqueGrec09) [inline app/grails-app/services/de/laser/SubscriptionService.groovy]: License = Vertrag?

File: app/grails-app/views/subscription/surveysConsortia.gsp [hunk 4]
```diff
@@ -67,7 +67,7 @@
                    value="${surveyConfig.pickAndChoose ? com.k_int.kbplus.SurveyOrg.findAllBySurveyConfigAndFinishDateIsNotNull(surveyConfig) : com.k_int.kbplus.SurveyResult.findAllBySurveyConfigAndFinishDateIsNotNull(surveyConfig)?.participant?.flatten()?.unique { a, b -> a.id <=> b.id }}"/>
 
             <g:set var="participantsTotal"
-                   value="${surveyConfig.pickAndChoose ?  com.k_int.kbplus.SurveyOrg.findAllBySurveyConfig(surveyConfig) : com.k_int.kbplus.SurveyResult.findAllBySurveyConfig(surveyConfig)?.participant?.flatten()?.unique { a, b -> a.id <=> b.id }}"/>
+                   value="${surveyConfig.pickAndChoose ? com.k_int.kbplus.SurveyOrg.findAllBySurveyConfig(surveyConfig) : com.k_int.kbplus.SurveyResult.findAllBySurveyConfig(surveyConfig)?.participant?.flatten()?.unique { a, b -> a.id <=> b.id }}"/>
```
Comment (croqueGrec09) [inline app/grails-app/views/subscription/surveysConsortia.gsp]: We should check whether we may win performance by avoiding flatten and unique and doing this by DBQuery instead

File: app/grails-app/views/templates/survey/_subscriptionSurvey.gsp [hunk 5]
```diff
@@ -232,13 +271,138 @@
...
+
                         <br>
                         <g:set var="derivedPropDefGroups"
                                value="${subscriptionInstance?.owner?.getCalculatedPropDefGroups(contextService.getOrg())}"/>
 
                         <div class="ui form">
                             <div class="two fields">
                                 <div class="eight wide field" style="text-align: left;">
+                                    <button id="subscription-properties-toggle"
+                                            class="ui button">Lizenzmerkmale anzeigen</button>
```
Comment (croqueGrec09) [inline app/grails-app/views/templates/survey/_subscriptionSurvey.gsp]: translation missing

File: app/grails-app/services/de/laser/SubscriptionService.groovy (hunk 3 above)
Comment (Moetez16) [reply app/grails-app/services/de/laser/SubscriptionService.groovy]: License = Lizenz

File: app/grails-app/conf/BootStrap.groovy (hunk 1 above)
Comment (Moetez16) [reply app/grails-app/conf/BootStrap.groovy]: YES

File: app/grails-app/views/templates/survey/_subscriptionSurvey.gsp (hunk 5 above)
Comment (croqueGrec09) [reply app/grails-app/views/templates/survey/_subscriptionSurvey.gsp]: Done in other, greater bunch

**Pull request merged**